### DISCUSSION TITLE
2.4.50 soup upgrade does not work

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#12458] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

24

### RAM

128

### Storage for /

500GB

### Storage for /nsm

6TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

i am using v2.4.40 and i want to upgrade to 2.4.50.
uploaded the iso file and running sudo soup -y -f <path to iso>. the upgrade begins but failes

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Do you see errors in the `soup.log`?

**Reply by udi-mosh** [reply]
looking at /root/soup.log...
i do not see any errors. it tells me that it found version 2.4.40 and it is an airgapped installation and salt is not at the latest and that i will need to run setup multiple times.
tried that prior to opening this case and i always get the same result. i get errors that are not in the log. errors that include IO (no problem with the disks)

**Reply by cm-ops** [reply]
Can you share the errors please?

**Reply by udi-mosh** [reply]
[so_error.txt](https://github.com/Security-Onion-Solutions/securityonion/files/14482566/so_error.txt)

**Reply by udi-mosh** [reply]
i have tried the checksum test and i see that what i have downloaded does not have the same checksum. downloading it now and will update.

**Reply by udi-mosh** [reply, marked as answer]
just finished downloading and it appears that the file i have downloaded before was not complete even though it seemed liked one.

**Comment by Phillip-Samson** [comment]
Is -f a valid flag? what happens when you use just -y

**Reply by udi-mosh** [reply]
-f is a must. it's a flag to the iso file
